### ISSUE TITLE
how many connections per thread?

### ISSUE THREAD

**Issue by cheers00** [wg/wrk#157]
In the following example, does each thread make 400/12 connections?

```
Basic Usage

  wrk -t12 -c400 -d30s http://127.0.0.1:8080/index.html

  This runs a benchmark for 30 seconds, using 12 threads, and keeping
  400 HTTP connections open.
```

**Comment by AlekSi** [comment]
See #90.
I would like to this in docs too.

**Comment by fatso83** [comment]
It's a connection pool. Nothing says it is an even split among the threads.

**Comment by AlekSi** [comment]
Nothing in the docs, right. But code in fact makes a split: https://github.com/wg/wrk/blob/master/src/wrk.c#L104

**Comment by fatso83** [comment]
Good catch, that is interesting. Then it seems that the remaining connections (c%t=r) are not used in any way. Setting up 15 connections with 8 threads would then just use 8 connections. So there should be a tip to use a number that is divisible for optimal resource usage.

**Comment by daluu** [comment]
Clarification would definitely help the greater community optimally use wrk w/o having to look through the source code or read through these issues to confirm behavior.

wrk2 makes it more complex by adding the rate parameter on top of threads & connections.

**Comment by AlekSi** [comment]
README was updated to say:
```
-c, --connections: total number of HTTP connections to keep open with
                   each thread handling N = connections/threads
```

@cheers00, I think this issue can be closed.
